src/test/java/seedu/guest/logic/commands/DeleteCommandTest.java

```diff
@@ -7,7 +7,7 @@
 import static seedu.guest.logic.commands.CommandTestUtil.showGuestAtIndex;
 import static seedu.guest.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
```

Comment:
Should we refactor constants in TypicalIndexes as well? e.g., INDEX_FIRST_GUEST, INDEX_SECOND_GUEST

Reply:
fixed

Reply:
On my end, the renaming of constants has not been fixed. Can I check if it has been updated on your end?